app/models/manageiq/providers/openstack/cloud_manager/event_catcher.rb

```diff
@@ -6,10 +6,6 @@ def self.ems_class
   end
 
   def self.all_valid_ems_in_zone
-    # Overridden to gate the execution of OpenstackEventMonitor in cases where
-    # the qpid-cpp-client-devel package is not available.  See
-    # PerEmsWorkerMixin#sync_workers and PerEmsWorkerMixin#start_workers to
-    # understand the overall flow when starting Event Catchers.
     require 'openstack/openstack_event_monitor'
```

Comment:
Not sure if we need this anymore ... I removed the comment because it talked specifically about the qpid-reasons for this method to exist.  But, it's possible we're relying on this in general when, say, the Rabbit connection fails, too.